vm/src/stdlib/posix.rs

```diff
@@ -1975,4 +1979,23 @@ pub mod module {
         res.map_err(|err| err.into_pyexception(vm))?;
         Ok(vm.ctx.new_int(written as u64).into())
     }
+
+    #[cfg(target_os = "linux")]
+    #[pyfunction]
+    fn getrandom(size: isize, flags: OptionalArg<u32>, vm: &VirtualMachine) -> PyResult<Vec<u8>> {
+        let size = usize::try_from(size)
+            .map_err(|_| vm.new_os_error(format!("Invalid argument for size: {}", size)))?;
+        let mut buf = vec![0u8; size];
+        let ret = unsafe {
+            libc::getrandom(
+                buf.as_mut_ptr() as *mut libc::c_void,
+                buf.len(),
+                flags.unwrap_or_else(|| 0),
```

Comment:
unwrap_or_else with const is same as just unwrap_or.
```suggestion
                flags.unwrap_or(0),
```

Reply:
Didn't know about this. Will update as suggested.